pkgs/top-level/python-packages.nix

```diff
@@ -17594,6 +17594,49 @@ in {
     };
   };
 
+  pypcap = buildPythonPackage rec {
```

Comment:
see the header of this file. This expression shouldn't be here.

Reply:
Sorry, I simply was overwhelmed by this 26k lines long file and ignored the head. Should be fixed now.